bin/peasant.js

```diff
@@ -25,6 +25,7 @@ var args = require('yargs')
   .command('link', 'Create symlinks text editor for linting')
   .command('lint', 'Source code error & style check')
   .command('build', 'Transpile sources for release')
+  .command('web', 'Transpile sources for web release')
```

Comment:
At the call-site, the command name `web` isn't very descriptive of what this command does. To me, this feels like it should be an option for `build`. Running `peasant build --web` would be clear at the call-site and I think makes more sense. Building for web is a subset of build.